Apps/WaitingQueueWeb/WaitingQueueWeb.csproj

```diff
@@ -8,7 +8,9 @@
 
     <PropertyGroup Condition=" '$(RunConfiguration)' == 'WaitingQueue' " />
     <ItemGroup>
+        <PackageReference Include="Hellang.Middleware.ProblemDetails" Version="6.5.1" />
```

Comment:
Can we confirm we need this after .NET 7 upgrade?

Reply:
This is still required or it will not compile.